circle.yml

```diff
@@ -145,8 +145,8 @@ jobs:
       - CC:  gcc-6
       - GCOV: gcov-6
       - GENERATOR: Ninja
-      - BUILD_PARALLEL_JOBS: 4
-      - TEST_PARALLEL_JOBS: 4
```

Comment:
This should not affect running tests in parallel

Reply:
Ok, I'll change this back